webpages/profile/metaanalysis.html

```diff
@@ -14,7 +14,7 @@
   <a href="/" id="logo-box">
     <h1>LiMA</h1><h2>Living Meta-Analysis</h2>
   </a>
-  
+
```

Comment:
In the real world - people will take offense to unnecessary white space fixes.

Typically, if you are not touching that line, leave it alone. The white space issues can be all dealt with in a specific "fix whitespace" commit.